src/main/events/helpers/validate-path.ts

```diff
@@ -0,0 +1,10 @@
+import fs from "fs";
+
+export default function validatePath(path: string): Error | undefined {
```

Comment:
Do not use `export default`, follow `export const` constraint.
Plus, do not use `function expression` unless you have to.